iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/NetworkSerDe.java

```diff
@@ -824,9 +832,9 @@ private static void readExtensions(Network network, String id, NetworkDeserializ
             if (context.getOptions().withExtension(extensionName) && !context.isIgnoredEquipment(id)) {
                 ExtensionSerDe<?, ?> extensionXmlSerializer = EXTENSIONS_SUPPLIER.get().findProvider(extensionName);
                 if (extensionXmlSerializer != null) {
-                    if (extensionXmlSerializer instanceof PostponableCreationExtensionSerDe<?, ?> extSerDe) {
-                        Function<Extendable<?>, ?> extensionCreator = (Function<Extendable<?>, ?>) extSerDe.readAndGetPostponableCreator(context);
-                        context.getEndTasks().add(() -> createExtension(network, id, extensionCreator, extensionNamesImported, extensionName));
+                    if (extensionXmlSerializer.postponeDeserialization()) {
+                        Function<Extendable<?>, ?> extensionCreator = (Function<Extendable<?>, ?>) extensionXmlSerializer.readAndGetPostponableCreator(context);
+                        context.addEndTask(DeserializationEndTask.Step.AFTER_EXTENSIONS, () -> createExtension(network, id, extensionCreator, extensionNamesImported, extensionName));
```

Comment:
I don't see in which case you still need to do this

Reply:
I forgot to remove them. It's now done.